Makefile

```diff
@@ -64,6 +67,19 @@ coverage: $(ACTIVATE)
 	@echo "Computing unit test coverage..."
 	@. $(ACTIVATE); coverage run --source=ie_serving -m pytest $(TEST_DIRS)/unit/ && coverage report --fail-under=70
 
+ams_image: $(AMS_EXAMPLE)requirements.txt $(AMS_EXAMPLE)requirements-dev.txt
```

Comment:
I think the name of this target should be something like `ams_test`

Reply:
Done.